packages/sihl-email/sihl-email.3.0.4/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Email service implementations for Sihl"
+description: "Modules for sending emails using Lwt and SMTP or Sendgrid."
+maintainer: ["josef@oxidizing.io"]
+authors: ["Josef Erben" "Aron Erben" "Miko Nieminen"]
+license: "MIT"
+homepage: "https://github.com/oxidizing/sihl"
+doc: "https://oxidizing.github.io/sihl/"
+bug-reports: "https://github.com/oxidizing/sihl/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08.0"}
+  "letters" {>= "0.2.1"}
```

Comment:
This requires a lower bound:
```
#=== ERROR while compiling sihl-email.3.0.4 ===================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.12.1 | pinned(https://github.com/oxidizing/sihl/archive/3.0.4.tar.gz)
# path                 ~/.opam/4.12/.opam-switch/build/sihl-email.3.0.4
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p sihl-email -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/sihl-email-7-ebc866.env
# output-file          ~/.opam/log/sihl-email-7-ebc866.out
### output ###
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -g -bin-annot -I sihl-email/src/.sihl_email.objs/byte -I /home/opam/.opam/4.12/lib/angstrom -I /home/opam/.opam/4.12/lib/asn1-combinators -I /home/opam/.opam/4.12/lib/astring -I /home/opam/.opam/4.12/lib/base -I /home/opam/.opam/4.12/lib/base/base_internalhash_types -I /home/opam/.opam/4.12/lib/base/caml -I /home/opam/.opam/4.12/lib/base/shadow_stdlib -I /home/opam/.opam/4.12/lib/base64 -I /home/opam/.opam/4.12/lib/base64/rfc2045 -I /home/opam/.opam/4.12/lib/bigarray-compat -I /home/opam/.opam/4.12/lib/bigarray-overlap -I /home/opam/.opam/4.12/lib/bigstringaf -I /home/opam/.opam/4.12/lib/bos -I /home/opam/.opam/4.12/lib/ca-certs -I /home/opam/.opam/4.12/lib/caqti -I /home/opam/.opam/4.12/lib/caqti-lwt -I /home/opam/.opam/4.12/lib/cmdliner -I /home/opam/.opam/4.12/lib/cohttp -I /home/opam/.opam/4.12/lib/cohttp-lwt -I /home/opam/.opam/4.12/lib/cohttp-lwt-unix -I /home/opam/.opam/4.12/lib/coin -I /home/opam/.opam/4.12/lib/colombe -I /home/opam/.opam/4.12/lib/colombe/emile -I /home/opam/.opam/4.12/lib/conduit -I /home/opam/.opam/4.12/lib/conduit-lwt -I /home/opam/.opam/4.12/lib/conduit-lwt-unix -I /home/opam/.opam/4.12/lib/conformist -I /home/opam/.opam/4.12/lib/containers -I /home/opam/.opam/4.12/lib/containers/monomorphic -I /home/opam/.opam/4.12/lib/cstruct -I /home/opam/.opam/4.12/lib/cstruct-sexp -I /home/opam/.opam/4.12/lib/digestif -I /home/opam/.opam/4.12/lib/domain-name -I /home/opam/.opam/4.12/lib/dune-build-info -I /home/opam/.opam/4.12/lib/duration -I /home/opam/.opam/4.12/lib/either -I /home/opam/.opam/4.12/lib/emile -I /home/opam/.opam/4.12/lib/eqaf -I /home/opam/.opam/4.12/lib/eqaf/bigstring -I /home/opam/.opam/4.12/lib/eqaf/cstruct -I /home/opam/.opam/4.12/lib/faraday -I /home/opam/.opam/4.12/lib/faraday-lwt -I /home/opam/.opam/4.12/lib/faraday-lwt-unix -I /home/opam/.opam/4.12/lib/fieldslib -I /home/opam/.opam/4.12/lib/fmt -I /home/opam/.opam/4.12/lib/fpath -I /home/opam/.opam/4.12/lib/gmap -I /home/opam/.opam/4.12/lib/hkdf -I /home/opam/.opam/4.12/lib/hmap -I /home/opam/.opam/4.12/lib/httpaf -I /home/opam/.opam/4.12/lib/httpaf-lwt-unix -I /home/opam/.opam/4.12/lib/ipaddr -I /home/opam/.opam/4.12/lib/ipaddr-sexp -I /home/opam/.opam/4.12/lib/ipaddr/unix -I /home/opam/.opam/4.12/lib/jwto -I /home/opam/.opam/4.12/lib/ke -I /home/opam/.opam/4.12/lib/letters -I /home/opam/.opam/4.12/lib/logs -I /home/opam/.opam/4.12/lib/lwt -I /home/opam/.opam/4.12/lib/lwt/unix -I /home/opam/.opam/4.12/lib/lwt_ssl -I /home/opam/.opam/4.12/lib/macaddr -I /home/opam/.opam/4.12/lib/magic-mime -I /home/opam/.opam/4.12/lib/mirage-crypto -I /home/opam/.opam/4.12/lib/mirage-crypto-ec -I /home/opam/.opam/4.12/lib/mirage-crypto-pk -I /home/opam/.opam/4.12/lib/mirage-crypto-rng -I /home/opam/.opam/4.12/lib/mirage-crypto-rng/lwt -I /home/opam/.opam/4.12/lib/mirage-crypto-rng/unix -I /home/opam/.opam/4.12/lib/mrmime -I /home/opam/.opam/4.12/lib/mtime -I /home/opam/.opam/4.12/lib/mtime/clock/os -I /home/opam/.opam/4.12/lib/multipart-form-data -I /home/opam/.opam/4.12/lib/nocrypto -I /home/opam/.opam/4.12/lib/ocaml-compiler-libs/common -I /home/opam/.opam/4.12/lib/ocaml-compiler-libs/shadow -I /home/opam/.opam/4.12/lib/ocaml/compiler-libs -I /home/opam/.opam/4.12/lib/ocaml/threads -I /home/opam/.opam/4.12/lib/ocplib-endian -I /home/opam/.opam/4.12/lib/ocplib-endian/bigstring -I /home/opam/.opam/4.12/lib/opium -I /home/opam/.opam/4.12/lib/parsexp -I /home/opam/.opam/4.12/lib/pbkdf -I /home/opam/.opam/4.12/lib/pecu -I /home/opam/.opam/4.12/lib/ppx_derivers -I /home/opam/.opam/4.12/lib/ppx_deriving/api -I /home/opam/.opam/4.12/lib/ppx_deriving/create -I /home/opam/.opam/4.12/lib/ppx_deriving/enum -I /home/opam/.opam/4.12/lib/ppx_deriving/eq -I /home/opam/.opam/4.12/lib/ppx_deriving/fold -I /home/opam/.opam/4.12/lib/ppx_deriving/iter -I /home/opam/.opam/4.12/lib/ppx_deriving/make -I /home/opam/.opam/4.12/lib/ppx_deriving/map -I /home/opam/.opam/4.12/lib/ppx_deriving/ord -I /home/opam/.opam/4.12/lib/ppx_deriving/runtime -I /home/opam/.opam/4.12/lib/ppx_deriving/show -I /home/opam/.opam/4.12/lib/ppx_deriving/std -I /home/opam/.opam/4.12/lib/ppx_deriving_yojson/runtime -I /home/opam/.opam/4.12/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.12/lib/ppxlib -I /home/opam/.opam/4.12/lib/ppxlib/ast -I /home/opam/.opam/4.12/lib/ppxlib/astlib -I /home/opam/.opam/4.12/lib/ppxlib/print_diff -I /home/opam/.opam/4.12/lib/ppxlib/stdppx -I /home/opam/.opam/4.12/lib/ppxlib/traverse_builtins -I /home/opam/.opam/4.12/lib/prettym -I /home/opam/.opam/4.12/lib/ptime -I /home/opam/.opam/4.12/lib/ptime/clock/os -I /home/opam/.opam/4.12/lib/re -I /home/opam/.opam/4.12/lib/re/str -I /home/opam/.opam/4.12/lib/result -I /home/opam/.opam/4.12/lib/rock -I /home/opam/.opam/4.12/lib/rosetta -I /home/opam/.opam/4.12/lib/rresult -I /home/opam/.opam/4.12/lib/safepass -I /home/opam/.opam/4.12/lib/sendmail -I /home/opam/.opam/4.12/lib/sendmail/starttls -I /home/opam/.opam/4.12/lib/seq -I /home/opam/.opam/4.12/lib/sexplib -I /home/opam/.opam/4.12/lib/sexplib0 -I /home/opam/.opam/4.12/lib/sihl -I /home/opam/.opam/4.12/lib/ssl -I /home/opam/.opam/4.12/lib/stdlib-shims -I /home/opam/.opam/4.12/lib/stringext -I /home/opam/.opam/4.12/lib/tls -I /home/opam/.opam/4.12/lib/tls/lwt -I /home/opam/.opam/4.12/lib/tsort -I /home/opam/.opam/4.12/lib/tyxml -I /home/opam/.opam/4.12/lib/tyxml/functor -I /home/opam/.opam/4.12/lib/unstrctrd -I /home/opam/.opam/4.12/lib/unstrctrd/parser -I /home/opam/.opam/4.12/lib/uri -I /home/opam/.opam/4.12/lib/uri-sexp -I /home/opam/.opam/4.12/lib/uri/services -I /home/opam/.opam/4.12/lib/uuidm -I /home/opam/.opam/4.12/lib/uutf -I /home/opam/.opam/4.12/lib/uuuu -I /home/opam/.opam/4.12/lib/x509 -I /home/opam/.opam/4.12/lib/yojson -I /home/opam/.opam/4.12/lib/yuscii -I /home/opam/.opam/4.12/lib/zarith -no-alias-deps -open Sihl_email__ -o sihl-email/src/.sihl_email.objs/byte/sihl_email.cmo -c -impl sihl-email/src/sihl_email.pp.ml)
# File "sihl-email/src/sihl_email.ml", line 150, characters 27-35:
# 150 |       Letters.Config.make ~username ~password ~hostname ~with_starttls
#                                  ^^^^^^^^
# Error: This expression has type string option
#        but an expression was expected of type string
```
Does it work with 0.3.0 or requires https://github.com/ocaml/opam-repository/pull/22002 ?